lib/mage/Mage.js

```diff
@@ -60,13 +71,18 @@ function Mage(mageRootModule) {
 
 	try {
 		rootPackage = require(rootPackagePath);
-	} catch (e) {}
+	} catch (error) {
```

Comment:
Fixes an issue where if your project's package.json is corrupted, it won't load and you will not be notified.